MPSGameLift/Documentation/GameLift.md

```diff
@@ -12,49 +12,18 @@ This README covers optional setup, testing and running on [Amazon GameLift](http
     C:\> aws --version
     aws-cli/2.10.0 Python/3.11.2 Windows/10 exe/AMD64 prompt/off
     ```
-1. Enable the "AWSGameLift" and "MPSGameLift" gems
-
-    Option 1: Commandline
-    ```sh
-    <path-to-o3de-engine>\scripts\o3de.bat enable-gem -pp <path-to-multiplayer-sample> -gn MPSGameLift
-    ```
-    Option 2: Project Manager
-    1. Open Project Manager
-    2. Select the "Configure Gems" options for Multiplayer Sample
-    3. Enable "AWSGameLift" and "MPSGameLift" gems
-
-        ![Enable GameLift Gems](Media/enable_gamelift_gems.jpg)
-    4. Click "Save"
-
-
-    ---
-    **NOTE**
-
-    Due to [bug-15829](https://github.com/o3de/o3de/issues/15829) please also add "AWSGameLift" to project.json's _gem_names_ list.
-
-    ---
-    
-
-1. Build the server, game launchers, and asset bundler for MultiplayerSample
-
-    `cmake --build build\windows --target Editor MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher AssetBundler --config profile -- /m `
-
-1. Build all the assets
-
-    `cmake --build build\windows --target MultiplayerSample.Assets --config profile -- /m`
-
 1. Work in progress (WiP) step: Add your AWS region to Config/default_aws_resource_mappings.json (example: "Region": "us-west-2")
 
     a. Currently needed otherwise when the client initializes GameLift there will be an error about not having a region.
 
     b. This step will be removed once we properly parse the game-session data which contains the fleet-id, region-id, etc  
 
-## Build Server for Windows
-1. Build Monolithic Server
+1. Use Export Project to Compile Code and Build Assets
```

Comment:
Do we want to add a disclaimer here regarding using an SDK flag for those using o3de release?

Reply:
Added an *important* note saying the scripting currently wont work with engine as sdk. I'll make a future pr